gnome-initial-setup/gis-driver.c

```diff
@@ -87,6 +87,14 @@ typedef struct _GisDriverPrivate GisDriverPrivate;
 G_DEFINE_TYPE_WITH_PRIVATE(GisDriver, gis_driver, GTK_TYPE_APPLICATION)
 
 static void
+scrolled_window_size_allocate (GtkScrolledWindow *sw,
+                               GdkRectangle      *alloc,
+                               GisDriver         *driver)
+{
+  gtk_adjustment_set_value (gtk_scrolled_window_get_vadjustment (sw), 0);
```

Comment:
This does not look entirely right:
- it should probably use `gtk_adjustment_get_lower()` instead of `0`
- it does not seem right to do this every size-allocate

Can you not do this only when the page is switched?